examples_test.go

```diff
@@ -19,3 +20,16 @@ func ExampleNewServer_basePath() {
 	}))
 	log.Fatal(http.ListenAndServe(":7643", nil))
 }
+
+type contextKey struct{ name string }
+
+var requestNoChange = contextKey{"RequestNoChange"}
+
+func tellRequestNoChange(r *http.Request) *http.Request {
+	return r.WithContext(context.WithValue(r.Context(), requestNoChange, struct{}{}))
+}
+
+func isRequestNoChange(r *http.Request) bool {
+	_, ok := r.Context().Value(requestNoChange).(struct{})
+	return ok
+}
```

Comment:
What does this do? Is it necessary?

Reply:
Deleted it, fixed test logic.
https://github.com/elimity-com/scim/pull/102/commits/ddb6180f6df83c7aac80a4ec3dec80e28fcfa8cd